fido-device-onboard.spec

```diff
@@ -0,0 +1,122 @@
+%define dracutlibdir %{_prefix}/lib/dracut
+%bcond_without check
+%global __cargo_skip_build 0
+%global __cargo_is_lib() false
+%global gitversion 43f7674dad4abc470cdab505fbf49f571539cea8
```

Comment:
Is there any chance that the one we add in the repo has the gitversion automatically determined?

Reply:
not sure I fully understand - this is also just for testing, we're gonna have semver versions here I think

Reply:
ok I understand - how would you do that tho? :thinking:

Reply:
we can template the spec and add a Makefile target that does that if we're ever going to support a `make rpm` target - ok I'll do that

Reply:
done, there's a `make rpm` now that builds the RPM and populates the commit variable as well